redis@7.2.4: download via aria2 failed

redis: 7.0.15 -> 7.2.4
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
WARN  To disable this warning, run 'scoop config aria2-warning-enabled false'.
Updating one outdated app:
Updating 'redis' (7.0.15 -> 7.2.4)
Downloading new version
Starting download with aria2 ...
Download: 03/21 16:58:10 [ERROR] CUID#8 - Download aborted. URI=https://github.com/redis-windows/redis-windows/releases/Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/redis-windows/redis-windows/releases/doDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 676e89|ERR |       0B/s|D:/Application/Scoop/cache/redis#7.2.4#https_github.com_redis-windows_redis-windows_releases_download_7.2.4_Redis-7.2.4-Windows-x64.tar.gz
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/redis-windows/redis-windows/releases/download/7.2.4/Redis-7.2.4-Windows-x64.tar.gz
    referer=https://github.com/redis-windows/redis-windows/releases/download/7.2.4/
    dir=D:\Application\Scoop\cache
    out=redis#7.2.4#https_github.com_redis-windows_redis-windows_releases_download_7.2.4_Redis-7.2.4-Windows-x64.tar.gz

ERROR & 'D:\Application\Scoop\apps\aria2\current\aria2c.exe' --input-file='D:\Application\Scoop\cache\redis.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=30772 --continue --summary-interval=0 --auto-save-interval=1

--- a/bucket/redis.json
+++ b/bucket/redis.json
@@ -5,9 +5,9 @@
     "license": "BSD-3-Clause",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/redis-windows/redis-windows/releases/download/7.2.4/Redis-7.2.4-Windows-x64.tar.gz",
-            "hash": "733a6611c8cb84c2de6d1355dc0392fd1f31d2a8731ca4a3eadbea98a9cd1ccc",
-            "extract_dir": "Redis-7.2.4-Windows-x64"
+            "url": "https://github.com/redis-windows/redis-windows/releases/download/7.2.4/Redis-7.2.4-Windows-x64-msys2.zip",
+            "hash": "a4c0ca3d7f91559a9cf8db3ae86aeb5362f67d91968109c317dc3e858ff87f82",
+            "extract_dir": "Redis-7.2.4-Windows-x64-msys2"
         }
     },
     "bin": [
@@ -21,8 +21,8 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/redis-windows/redis-windows/releases/download/$version/Redis-$version-Windows-x64.tar.gz",
-                "extract_dir": "Redis-$version-Windows-x64"
+                "url": "https://github.com/redis-windows/redis-windows/releases/download/$version/Redis-$version-Windows-x64-msys2.zip",
+                "extract_dir": "Redis-$version-Windows-x64-msys2"
             }
         }
     }

